web/packages/plugins/shared/common/src/components/UploadFileButton.tsx

```diff
@@ -56,7 +55,7 @@ export function UploadFileButton(props: {
       setError(`Only files of type '${fileSuffix}' can be uploaded here`)
     } else {
       const newDocumentBody = getBody(file.name)
-      addToPath(newDocumentBody, token, [file], dataSourceId)
+      addToPath(newDocumentBody, token, [file], dataSourceId, 'Data/STasks')
```

Comment:
Should Data/Stasks be hardcoded here?

Reply:
It already was the default value for this function. It's where STasks are uploaded.